.github/workflows/continuous-integration.yml

```diff
@@ -2,23 +2,25 @@ name: "Continuous Integration"
 
 on:
   pull_request:
+    branches:
+      - "*.x"
+    paths:
+      - .github/workflows/continuous-integration.yml
+      - composer.*
+      - lib/**
+      - phpunit.xml.dist
+      - tests/**
   push:
     branches:
       - "*.x"
+    paths:
+      - .github/workflows/continuous-integration.yml
+      - composer.*
+      - lib/**
+      - phpunit.xml.dist
+      - tests/**
 
 jobs:
-  composer_normalize:
-    name: "Composer Normalize"
-    runs-on: "ubuntu-22.04"
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-
-      - uses: "docker://ergebnis/composer-normalize-action:latest"
```

Comment:
That image hasn't receive a push in 2 years, which might explain why the build is broken :P